From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Juan Sanchez <juanesf91@gmail.com>
Date: Thu, 30 Jul 2026 10:00:00 -0400
Subject: [PATCH] dt-bindings: clock: sun55i a523 mcu ccu: add CLK_BUS_MCU_DSP

Add the clock ID for the DSP bus clock gate at MCU CCU register 0x0100 BIT(1).

Signed-off-by: Juan Sanchez <juanesf91@gmail.com>
---
 include/dt-bindings/clock/sun55i-a523-mcu-ccu.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/dt-bindings/clock/sun55i-a523-mcu-ccu.h b/include/dt-bindings/clock/sun55i-a523-mcu-ccu.h
index 6efc6bc7e..ce2b3ceab 100644
--- a/include/dt-bindings/clock/sun55i-a523-mcu-ccu.h
+++ b/include/dt-bindings/clock/sun55i-a523-mcu-ccu.h
@@ -50,5 +50,6 @@
 #define CLK_BUS_MCU_RISCV_MSGBOX	41
 #define CLK_MCU_PWM0		42
 #define CLK_BUS_MCU_PWM0	43
+#define CLK_BUS_MCU_DSP		44
 
 #endif /* _DT_BINDINGS_CLK_SUN55I_A523_MCU_CCU_H_ */
